packages/digitalproperty-app/lib/cmds/landregistry_cmds/addLandTitles.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+const LandRegistry = require('./../../landRegistry.js');
+const winston = require('winston');
+const LOG = winston.loggers.get('application');
+
+
+
+
+
+module.exports.command = 'addLandTitle [options]';
+module.exports.desc = 'Add a land title to the registry.';
+module.exports.builder = function(yargs) {
+
+    return yargs
+    
+    .option('p', { alias: 'PID', describe: 'Please type in your PersonID.', type: 'string'})
```

Comment:
These descriptions come out on the command line - so not in an interactive mode. So probably best phrased more as facts.

Have a look at some of the other help messages for examples.